target/linux/ramips/dts/mt7621_zte_mf286c3.dts

```diff
@@ -0,0 +1,316 @@
+			function-enumerator = <8>;
+			gpios = <&aw9523 3 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-10 { /* p1_1 - sig11 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <9>;
+			gpios = <&aw9523 9 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <50>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&button_pins>;
+
+		key-wps {
+			gpios = <&aw9523 14 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		key-wlan {
+			gpios = <&aw9523 15 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WLAN>;
+		};
+	};
+};
+
+&nand {
+	status = "okay";
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "Bootloader";
+			reg = <0x0 0x80000>;
+			read-only;
+		};
+
+		partition@80000 {
+			label = "Config";
+			reg = <0x80000 0x80000>;
+			read-only;
+		};
+
+		partition@100000 {
+			label = "Factory";
+			reg = <0x100000 0x100000>;
+			read-only;
+
+			nvmem-layout {
+				compatible = "fixed-layout";
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				eeprom0: eeprom@0 {
+					reg = <0x0 0x400>;
+				};
+
+				eeprom1: eeprom@8000 {
+					reg = <0x8000 0x4da8>;
+				};
+
+				macaddr_factory_4: macaddr@4 {
+					reg = <0x4 0x6>;
+				};
+
+				macaddr_factory_8004: macaddr@8004 {
+					reg = <0x8004 0x6>;
+				};
+
+				macaddr_factory_e000: macaddr@e000 {
+					reg = <0xe000 0x6>;
+				};
+
+				macaddr_factory_e006: macaddr@e006 {
+					reg = <0xe006 0x6>;
+				};
+			};
+		};
+
+		partition@200000 {
+			label = "inventory";
+			reg = <0x200000 0x100000>;
+			read-only;
+		};
+
+		partition@300000 {
+			label = "customer";
+			reg = <0x300000 0x100000>;
+			read-only;
+		};
+
+		partition@400000 {
+			label = "zone";
+			reg = <0x400000 0x100000>;
+			read-only;
+		};
+
+		partition@500000 {
+			label = "reserved0";
+			reg = <0x500000 0x500000>;
+			read-only;
+		};
+
+		partition@a00000 {
+			label = "kernel";
+			reg = <0xa00000 0x500000>;
+		};
+
+		partition@f00000 {
+			label = "ubi";
+			reg = <0xf00000 0x7100000>;
+		};
+	};
+};
+
+&i2c {
+	status = "okay";
+	aw9523: gpio-expander@5b {
+		compatible = "awinic,aw9523-pinctrl";
+		reg = <0x5b>;
+		gpio-controller;
+		#gpio-cells = <2>;
+		gpio-ranges = <&aw9523 0 0 16>;
+
+		reset-gpios = <&gpio 6 GPIO_ACTIVE_HIGH>;
+
+		button_pins: button-pins {
+			pins = "gpio4", "gpio7", "gpio13", "gpio14";
+			function = "gpio";
+			input-enable;
+		};
+
+		led_pins: led-pins {
+			pins = "gpio0", "gpio1", "gpio2", "gpio3", "gpio5", "gpio6",
+				"gpio8", "gpio9", "gpio10", "gpio11", "gpio12";
+			function = "gpio";
+			input-disable;
+			output-low;
+		};
+	};
+};
+
+&pcie {
+	status = "okay";
+};
+
+&pcie0 {
+	wifi@0,0 {
+		compatible = "mediatek,mt76";
+		reg = <0x0 0 0 0 0>;
+		ieee80211-freq-limit = <2400000 2500000>;
+		nvmem-cells = <&eeprom0>, <&macaddr_factory_4>;
```

Comment:
`, <&macaddr_factory_4>` can be removed as we don't need it now.

Reply:
`, <&macaddr_factory_4>;` is still there why is this marked as solved?